scripts/shared/lib/utils

```diff
@@ -105,12 +101,33 @@ function declare_cidrs() {
     # shellcheck disable=SC2034 # these variables are used elsewhere
     declare -gA cluster_CIDRs service_CIDRs global_CIDRs
 
-    for i in {1..3}; do
-        add_cluster_cidrs $i
+    i=1
+    for cluster in $(eval echo "$1"); do
+        add_cluster_cidrs "$i" "$cluster"
+        i=$(("$i"+1))
     done
 }
 
 function declare_kubeconfig() {
+    local clusters
+    clusters=$(kind get clusters)
     export KUBECONFIG
-    KUBECONFIG=$(echo "${KUBECONFIGS_DIR}"/kind-config-cluster{1..3} | sed 's/ /:/g')
+    KUBECONFIG=
+    for cluster in ${clusters}; do
+        [ -n "${KUBECONFIG}" ] && KUBECONFIG+=":" 
+        KUBECONFIG+="${KUBECONFIGS_DIR}/kind-config-${cluster}"
+    done
+}
+
+function print_clusters_message() {
+    num_clusters=$(kind get clusters | wc -l)
```

Comment:
I wouldn't use this since you could have other kind clusters not related... You can just not print the number, or alternatively you could count the number of kubeconfig files.

Reply:
Wouldn't counting the number of kubeconfig files reveal the same # as `kind get clusters`?  I'm fine with omitting the number.

Reply:
Not necessarily, anyway just omit it it'll be the simplest :)